great_expectations/expectations/expectation.py

```diff
@@ -312,6 +312,15 @@ def schema_extra(schema: Dict[str, Any], model: Type[Expectation]) -> None:
                 },
             }
 
+            # Add extra fields to schema from custom schema_overrides
+            # schema_overrides is not a pydantic concept, but pydantic.Field allows
+            # us to pass through arbitrary fields.
+            for prop in schema["properties"].values():
+                if overrides := prop.pop("schema_overrides", None):
+                    assert isinstance(overrides, dict)
+                    for key, value in overrides.items():
+                        prop[key] = value
```

Comment:
This is fine, but cleaner might be:
```suggestion
                    prop.update(overrides)
```

Reply:
Oh yeah, much better.